src/client/components/EventsPage/EventsPage.component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { StudyGroupLogo } from '../StudyGroupLogo/StudyGroupLogo';
+import './EventsPage.styles.css';
+
+function EventsPage() {
+  return (
+    <div>
+      <div>Study Groups for Class15</div>
```

Comment:
`Class15` should come through props. Different students will have different text here.